runtime/oti/j9nonbuilder.h

```diff
@@ -3585,6 +3585,28 @@ typedef struct J9Method {
 	U_8* bytecodes;
 	struct J9ConstantPool* constantPool;
 	void* methodRunAddress;
+
+	/**
+	 * This field is overloaded and can have several meanings:
+	 * 
+	 * 1. If the `J9_STARTPC_NOT_TRANSLATED` bit is set, this method is interpreted
+	 * 		a. If the value is non-negative, then the field contains the invocation count which is decremented towards
+	 * 		zero. When the invocation count reaches zero, the method will be queued for JIT compilation. The invocation
+	 * 		count is represented as:
+	 * 
+	 * 		```
+	 * 		(invocationCount << 1) | J9_STARTPC_NOT_TRANSLATED
+	 * 		```
+	 * 
+	 * 		b. If the value is negative, then it can be one of:
+	 * 			- `J9_JIT_NEVER_TRANSLATE`: ???
+	 * 			- `J9_JIT_QUEUED_FOR_COMPILATION`: The method has been queued for JIT compilation
+	 * 			- `-1`: ???
+	 * 			- `0xCC`: ???
```

Comment:
We see this happening here:
https://github.com/eclipse/openj9/blob/250b180e1fa101e3dd3e171a3bb686b9301eab08/runtime/codert_vm/decomp.cpp#L261-L279

@gacholio could you elaborate on why we do this? It seems x86 specific, but the code is common to all platforms.

Reply:
Note that this code is not setting the extra field - it's using it as the jitted start address. I believe the documentation already there is sufficient explanation.

Reply:
You're right. We are overwriting the JIT entry instruction with 0xCC, not actually storing 0xCC to the `extra` field. I'll remove this from the documentation above.